rebasehelper/spec_hooks/pypi_url_fix.py

```diff
@@ -26,16 +26,19 @@
 
 
 class PyPIURLFixHook(BaseSpecHook):
-    """
-    SpecHook for transforming old nonfunctional python package url to
-    a safe url(eg. https://pypi.* to https://files.pythonhosted.*)
+    """SpecHook for transforming an old python package url to a new url(eg. https://pypi.* to
+    https://files.pythonhosted.* or https://pypi.python.org/pypi/* to https://pypi.org/project/*)
```

Comment:
I'd rather write that as `https://files.pythonhosted.org/packages/*` instead of `https://files.pythonhosted.*` - that's just too generic.